Bindings/bindings.json

```diff
@@ -270,6 +270,26 @@
                     ],
                     "label": "WebHook type",
                     "help": "WebHooks allow services to call one another easily. If the type you need isn't listed here, choose 'Not a WebHook'."
+                },
+                {
+                    "name": "authLevel",
+                    "value": "enum",
+                    "enum": [
+                        {
+                            "value": "anonymous",
+                            "display":"Anonymous"
+                        },
+                        {
+                            "value": "function",
+                            "display":"Function"
+                        },
+                        {
+                            "value": "admin",
+                            "display": "Admin"
+                        }
+                    ],
+                    "label": "Authorization level",
+                    "help": "Authorization level controls whether you're using an API key and which key to use. The function & master keys are found in your host.json. For user-based authentication, go to Function App Settings."
```

Comment:
Good point. I've updated the new one. Default is the "first" value, but we've found a cool bug here, so it might change.